site/docs/v1/tech/core-concepts/_email-templates.adoc

```diff
@@ -0,0 +1,44 @@
+[.api]
```

Comment:
We should identify which email templates will require a licensed version of FusionAuth, specifically with the threat detection feature.